Gemfile

```diff
@@ -2,6 +2,8 @@ source 'https://rubygems.org'
 
 gemspec
 
+gem 'sinatra', '< 2' if RUBY_VERSION < '2.2'
```

Comment:
This duplicates sinatra dependency declared in gemspec file (https://github.com/theforeman/smart-proxy/blob/develop/smart_proxy.gemspec#L18). Could you remove it from there, but keep it here?